python/src/scipp/plotting/resampling_model.py

```diff
@@ -96,23 +97,26 @@ def _call_resample(self):
                 low, high = s
                 if isinstance(low, int):
                     out = out[dim, low:high]
-                    low = out.meta[dim][dim, 0]
-                    high = out.meta[dim][dim, -1]
+                    low = out.coords[dim][dim, 0]
+                    high = out.coords[dim][dim, -1]
                 else:
                     # cannot pre-select bins for multi-dim coords
-                    if len(out.meta[dim].dims) == 1:
-                        out = out[sc.get_slice_params(out.data, out.meta[dim],
-                                                      low, high)]
+                    if len(out.coords[dim].dims) == 1:
+                        # TODO: for now, slicing in scipp does not support step
+                        sl = sc.get_slice_params(out.data, out.coords[dim],
+                                                 low, high)
+                        out = out[sl[0], slice(sl[1].start, sl[1].stop)]
```

Comment:
This is necessary because `get_slice_params()` always returns a slice with a step of 1, because `py::slice` wants the step to be specified: `py::slice(start, stop, 1)`.
It won't compile with `py::slice(start, stop)`.

Reply:
Are you saying that `__getitem__` fails if you specify a step, even if it is 1?